tensorboard/components/tf_backend/baseStore.ts

```diff
@@ -16,10 +16,17 @@ namespace tf_backend {
 
 export type Listener = () => void;
 
+export class ListenKey {
```

Comment:
Maybe add a short comment on what purpose this wrapper serves?  I assume it's somehow useful to have a distinct object representing the specific listener instance but I'm not quite sure.

Also, optional, but "handle" might be slightly clearer than "key" since "key" to me sounds more like what you'd use in a mapping where you might have different listeners for a given key, but here the point is really to be a unique wrapper/handle object for a unique listener.

Reply:
This term, ListenKey, is what Closure uses so I thought it would be more relatable this way. I can change if you feel strongly about it though.

Reply:
Nah, it's fine to leave it as ListenKey if there's precedent (just one that I'm not familiar with).